tests/src/test/v1.2/basic/fdc3.raiseIntentForContext.ts

```diff
@@ -10,8 +10,8 @@ const docs =
   "\r\nCause: ";
 
 export default () =>
-  describe("fdc3.raiseIntentForContext", async () => {
-    it("(BasicRI1) Method is callable", async () => {
+  describe("(NoContext) fdc3.raiseIntentForContext", async () => {
```

Comment:
In the sheet and in the 1.2 Test Definitions the (NoContext) test is part of the Intents section.  If we're going to change it here we also need to change the Test Definition PR in the FDC3 project.  

I suggest you keep the test where it was if possible so we don't have to do that work, even if it makes BasicRI1 a subset of NoContext.

Reply:
https://github.com/finos/FDC3/pull/848

Reply:
Ok, I'll move it back. In that case, I think we can remove the basic raiseIntentForContext.ts as the test within is a duplicate

Reply:
I've added that back in where it was

Reply:
nearly right - see my review